playground/ParameterEndToEnd/ParameterEndToEnd.AppHost/Program.cs

```diff
@@ -21,9 +21,12 @@
 
 var insertionrows = builder.AddParameter("insertionrows");
 
+var cs = builder.AddConnectionString("cs", $"sql={db};rows={insertionrows}");
```

Comment:
I like this and I don't like this at the same time. Obviously it looks nice and clean. But I wonder if this is confusing between "is this an interpolated string? or is this a ReferenceExpression?".

Reply:
We don't *need* to add this overload its a convenience.